exonum-java-binding-core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -102,6 +103,20 @@ impl JavaServiceRuntime {
     }
 }
 
+/// Returns path to <ejb-app location>/lib directory in an absolute form.
+fn absolute_library_path() -> String {
+    let library_path = {
+        // Get current path to EJB App.
+        let mut exe_location = env::current_exe().expect("Could not get the executable location");
```

Comment:
Can we move this sequence of operations:
```
env.current_exe().pop()
```
to a helper function? I believe there is a similar sequence in the code determining the system classpath.

Reply:
Probaly good idea, I thought about it